script.bluetooth.delay/addon.py

```diff
@@ -99,24 +79,32 @@ def speakers():
 def main():
 
     if arg == "reset":
-        xbmc.executebuiltin('PlayerControl(stop)')
-        Addon.setSettingBool('state', 1)
-        reset1()
+        if (xbmc.getCondVisibility('Player.HasVideo') == False):
+            xbmc.executebuiltin("PlayMedia("+path+"reset.mp4,1)")
+        xbmc.executebuiltin('ActivateWindow(busydialognocancel)')
+        time.sleep(2)
```

Comment:
Use xbmc.sleep() for small values, use XBMC.Monitor().waitForAbort() for longer values (usually > 4secs)